src/tests/Logary.Adapters.Facade.Tests/Program.fs

```diff
@@ -39,6 +40,8 @@ let stubLogManager (message: Message ref) =
       member x.shutdown (fDur,sDur) =
         Alt.always (FlushInfo([],[]),ShutdownInfo([],[]))
       member x.switchLoggerLevel (path, minLevel) = ()
+      member x.wrapScope scope = ()
```

Comment:
What does wrap scope do?

Reply:
aspnet core has its own external scope provider, when aspnet core logging framework `SetScopeProvider`, we wrap it from our own scope.  

https://github.com/logary/logary/blob/aspnet-logging/src/Logary.AspNetCore/LoggerProvider.fs#L14-L23

```fsharp
  interface ISupportExternalScope with
    member x.SetScopeProvider (scopeProvider: IExternalScopeProvider) =
      m.wrapScope {
        new ILogScope with
          member x.collect () = 
            let mutableStateList = new ResizeArray<obj>()
            do scopeProvider.ForEachScope(new Action<obj,ResizeArray<obj>>(fun data state -> state.Add(data)), mutableStateList)
            mutableStateList |> List.ofSeq
          member x.push lazyData = scopeProvider.Push(lazyData.Value)
          member x.wrap scope = ()
} 
```

with this, other lib use aspnet logging abstraction api logging scoped value can be inject into logary too.